pkgs/top-level/python-packages.nix

```diff
@@ -6856,6 +6856,8 @@ self: super: with self; {
 
   mitmproxy = callPackage ../development/python-modules/mitmproxy { };
 
+  mitmproxy-rs = callPackage ../development/python-modules/mitmproxy-rs { };
+
   mitmproxy-wireguard = callPackage ../development/python-modules/mitmproxy-wireguard { };
```

Comment:
Dropping that in another PR. To quote the PR:

https://togithub.com/mitmproxy/mitmproxy/pull/5909

> mitmproxy-wireguard is already shipping in 9.0, but we will add more non-WireGuard stuff written in Rust.